### ISSUE TITLE
Using childnet_generic_mod.py for final training

### ISSUE THREAD

**Issue by rpand002** [hpnair/18663_Project_FBNet#2]
Hi, many thanks for your codes. I am able to search the architectures using train_cifa10.py but when I am using childnet_generic_mod.py with the theta_f, its not converging on cifar10 dataset. The accuracy is coming around 10%. I have even tried training with the model30_epoch_89.txt but the accuracy is still around 10%. Can you please let me know what you did to get the final accuracy on cifar10 dataset after searching the architecture? Thanks.

**Comment by rpand002** [comment]
@hpnair @SaiVineethKalluruSrinivas Can you please share your suggestion?

**Comment by SaiVineethKalluruSrinivas** [comment]
Hi @rpand002 
Yes, all models trained do not converge. Few of them do converge, which was possible by adding dropout and batchnorm layers. The position of these layers vary with different childnets. Feel free to explore hyperparameters while training childnets and even the layer (dropout and BN) positions. 

We would be closing this issue as we do not have any more related suggestions to provide.
